include/swift/AST/DiagnosticsSema.def

```diff
@@ -844,6 +844,9 @@ NOTE(unspaced_binary_operators_candidate,none,
 ERROR(unspaced_unary_operator,none,
       "unary operators must not be juxtaposed; parenthesize inner expression",
       ())
+ERROR(unexisting_power_operator,none,
```

Comment:
"Unexisting" is not an English word. Do you mean "nonexistent"?

Reply:
Thanks! I changed it to `nonexistent_power_operator`